Makefile

```diff
@@ -186,10 +194,19 @@ uninstall: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
-deploy: $(TOOLBIN)/kustomize
+deploy: $(TOOLBIN)/kustomize update-version
 	cd config/kube-app-manager && $(TOOLBIN)/kustomize edit set image controller=$(CONTROLLER_IMG)
 	$(TOOLBIN)/kustomize build config/default | $(TOOLBIN)/kubectl apply -f -
 
+.PHONY: update-version
+update-version:
+	sed -i ''  "s|app.kubernetes.io/version: dev|app.kubernetes.io/version: $(VERSION)|g" config/default/kustomization.yaml
```

Comment:
this seems like an anti-pattern. and not sure if there is a better way.

https://github.com/kubernetes-sigs/kustomize/issues/388
@Liujingfang1 FYI

Reply:
A better way of doing this is to create a new kustomization target for that version. For example, use `config/v1.0.0` directory for the version `v1.0.0`.

Reply:
Thanks for your recommendation @Liujingfang1

Reply:
We can try this way(new kustomization target) as suggested, and discuss later if it becomes too much work